README.md

```diff
@@ -1,6 +1,6 @@
 # python-bip32
 
-A basic implementation of the [bip-0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki).
+A basic implementation of [BIP-0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki).
```

Comment:
I prefer using either "BIP32"/"BIP 32" or "bip-0032". But this diff is already an improvement over the current situation, so this will do for now.